### PR TITLE
hour of code events text update

### PR DESCRIPTION
**Pull request by dmantonyuk**
Update description at top of hourofcode.com/events

## Links

https://codedotorg.atlassian.net/browse/ACQ-1132

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by dmantonyuk** [comment]
hour of code events

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 1]
```diff
@@ -1658,6 +1658,7 @@
 
   hoc_homepage_join_the_movement_heading: 'Join the movement'
   hoc_homepage_join_the_movement_description: 'Hour of Code is available year-round, but every year in December your class can join millions of students around the world celebrating Computer Science Education Week with the Hour of Code. Registration for the annual celebration starts each year in October.'
+  hoc_homepage_join_the_celebration_description: 'Your class or group can join millions around the world doing the Hour of Code! This year, the annual Computer Science Education Week celebration is December 4-10 and registration begins in October. Though, Hour of Code is available year-round.'
```
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: The dates here need to be dynamic. You can replace this string with: 
`Your class or group can join millions around the world doing the Hour of Code! This year, the annual Computer Science Education Week celebration is %{campaign_date} and registration begins in October. Though, Hour of Code is available year-round.`

File: pegasus/sites.v3/hourofcode.com/views/signup_form.haml [hunk 2]
```diff
@@ -10,7 +10,7 @@
   %h2#signup-header= hoc_s(:signup_header, locals: {campaign_date: campaign_date("full")})
   %h2#census-header{style: "display: none;"}= hoc_s(:census_header)
   %p.body-two
-    =hoc_s(:hoc_homepage_join_the_movement_description)
+    =hoc_s(:hoc_homepage_join_the_celebration_description)
```
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/views/signup_form.haml]: Once the string has the dynamic date variable, you can replace this with:
`=hoc_s(:hoc_homepage_join_the_celebration_description), markdown: :inline, locals:{campaign_date: campaign_date("full")})`

**Review by kelbyhawn — COMMENTED** [review]
Looks good, except the date for CSEdWeek needs to be dynamic. I left comments on how to update this.

**Review by kelbyhawn — APPROVED** [review]
Looks great!

**Pull request merged**